app/views/pages/dashboard.html.erb

```diff
@@ -1,5 +1,5 @@
 <div class="container justify-content-center my-3">
-  <h1>DASHBOARD</h1> <h2>"Hey <%= @user.name %>"</h2>
+  <h1>DASHBOARD</h1> <h2>"Hey <%= @username %>"</h2>
```

Comment:
Hey @carolyne-web , this should be @user.username

Reply:
I'll change it :)

Reply:
thank you!!